### ISSUE TITLE
llamas test

### ISSUE THREAD

**Issue by wlp1213** [voldemortX/pytorch-auto-drive#178]
Hi, how can I obtain the final F1 score when the test set labels are not provided in llamas? After running
python main_landet.py --test --config=configs/lane_detection/bezierlanenet/resnet18_llamas_aug1b.py --mixed-precision
there is no final result in log.txt.

**Comment by voldemortX** [comment]
> Hi, how can I obtain the final F1 score when the test set labels are not provided in llamas? After running
> python main_landet.py --test --config=configs/lane_detection/bezierlanenet/resnet18_llamas_aug1b.py --mixed-precision
> there is no final result in log.txt.
> 

@wlp1213 I remember the llamas test results can only be obtained in their online server?

**Comment by wlp1213** [comment]
Thank you very much for your response. Additionally, I encountered the following issue when obtaining the results for the llamas validation set. 
![y](https://github.com/user-attachments/assets/d3fbb76d-aafa-4ffe-9a77-ab81d69005a6)
There were no issues with training and predicting lane points for testing, but an error occurred during the test.

**Comment by voldemortX** [comment]
> Thank you very much for your response. Additionally, I encountered the following issue when obtaining the results for the llamas validation set. 
> ![y](https://github.com/user-attachments/assets/d3fbb76d-aafa-4ffe-9a77-ab81d69005a6)
> There were no issues with training and predicting lane points for testing, but an error occurred during the test.

It seems to be a json format issue. You sure the dataset is correctly downloaded?

**Comment by wlp1213** [comment]
Okay, thank you once again for your response.
